docs/contributing.md

```diff
@@ -25,6 +25,7 @@ To contribute, you would need to fork the Hudi code on Github & then clone your
 
 We have embraced the code style largely based on [google format](https://google.github.io/styleguide/javaguide.html). Please setup your IDE with style files from [here](https://github.com/apache/incubator-hudi/tree/master/style).
 These instructions have been tested on IntelliJ. We also recommend setting up the [Save Action Plugin](https://plugins.jetbrains.com/plugin/7642-save-actions) to auto format & organize imports on save. The Maven Compilation life-cycle will fail if there are checkstyle violations.
+If you face jetty version related issues while running test cases, we recommend you to add spark jars to the classpath of your module in Intellij. 
```

Comment:
Can we keep this generic and avoid mentioning specific issues...  Ideally, we can redo this as bullets. and give specific instructions on how to add spark-jars to class path and leave it there?

Reply:
Done with the changes @vinothchandar